index.html

```diff
@@ -10,10 +10,28 @@
   <link rel="stylesheet" href="css/index.css">
 
 </head>
-
 <body>
+    <header>
+        <div class="topimg">
+            <img src="https://i.ibb.co/7b2ByCZ/lambda-black.png">
+        </div>
+
+        <!-- Nav Bar -->
+        <nav class="topnav">
```

Comment:
Since all of the anchor elements are the same we do not need to specify a separate class for each anchor element.  we can target it by nav a { }.